impl/client/src/main/javascript/web/pentaho/data/filter/standard.js

```diff
@@ -22,11 +22,15 @@ define([
   "./not",
   "./isEqual",
   "./isIn",
+  "./isGreater",
+  "./isLess",
+  "./isGreaterOrEqual",
+  "./isLessOrEqual",
   "./true",
   "./false"
 ], function(
-    abstractFactory, treeFactory, propFactory, andFactory, orFactory, notFactory, isEqFactory, isInFactory,
-    trueFactory, falseFactory) {
+    abstractFactory, treeFactory, propFactory,
+    andFactory, orFactory, notFactory, isEqFactory, isInFactory, isGtFactory, isLtFactory, isGteFactory, isLteFactory, trueFactory, falseFactory) {
```

Comment:
Can we have the same names for the factories everywhere? 
Since we already have `isGreaterFactory` in some files I would prefer to have it that way instead of `isGtFactory`.